frontend/packages/console-app/src/extensions/dashboards.ts

```diff
@@ -0,0 +1,63 @@
+import { Extension, LazyLoader } from '@console/plugin-sdk';
+import {
+  FirehoseResource,
+  FirehoseResourcesResult,
+} from '@console/internal/components/utils/types';
+import { K8sResourceCommon, K8sKind } from '@console/internal/module/k8s';
+import { HealthState } from '@console/shared/src/components/dashboard/status-card/states';
+
+namespace ExtensionProperties {
+  export interface DashboardsOperatorStatus<R extends K8sResourceCommon> {
```

Comment:
Added fallback to `K8sResourceCommon`
In case of URL subsystem, the `R` represents the response of a call to the given URL which can be anything like JSON or string, so i think that no fallback here is ok

Reply:
Right, I forgot that URL subsystem response can be anything.